File: Desafios em C#/1171 - Frequência de Números.cs
using System;
using System.Collections.Generic;

public class main {

	static void Main(string[] args) { 
	  
        int valores = int.Parse(Console.ReadLine());
        SortedDictionary<int, int> numeros = new SortedDictionary<int, int>();

        for (int i = 0; i < valores; i++) {
            var numero = int.Parse(Console.ReadLine());

            if(numeros.ContainsKey(numero)) {
               numeros[numero] += 1;
            } else {
                numeros.Add(numero, 1);
            }
        }

        foreach (KeyValuePair<int, int> n in numeros) {
            Console.WriteLine("{0} aparece {1} vez(es)", n.Key, n.Value);
        }

    }
}

File: Desafios em JavaScript/1171 - Frequência de Números.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

const valores = parseInt(lines[0]);
const numeros = new Map();

for (let i = 1; i <= valores; i++) {
  const numero = parseInt(lines[i]);

  if (numeros.has(numero)) {
    numeros.set(numero, numeros.get(numero) + 1);
  } else {
    numeros.set(numero, 1);
  }
}

Array.from(numeros.entries())
  .sort((a, b) => a[0] - b[0])
  .forEach((entry) => {
    console.log(`${entry[0]} aparece ${entry[1]} vez(es)`);
  });
